src/platforms/dialogflow/directives/facebook.ts

```diff
@@ -112,3 +112,119 @@ export class FacebookSuggestionChips implements IDirective {
     };
   }
 }
+
+export interface IFacebookQuickReply {
+  imageUrl: string;
+  title: string;
+  payload: string;
+}
+
+export class FacebookQuickReplyLocation implements IDirective {
+  public static platform: string = "dialogflow";
+  public static key: string = "facebookQuickReplyLocation";
+
+  constructor(public message: string) {}
+
+  public async writeToReply(
+    reply: IVoxaReply,
+    event: IVoxaEvent,
+    transition: ITransition,
+  ): Promise<void> {
+    const dialogFlowReply = (reply as DialogFlowReply);
+
+    dialogFlowReply.source = "facebook";
```

Comment:
seems like most of this directives are similar enough that we can create a helper for that, something like https://github.com/VoxaAI/voxa/blob/master/src/platforms/dialogflow/directives/assistant.ts#L64

Please explora that optino